lighthouse-cli/test/cli/run-test.js

```diff
@@ -25,6 +25,7 @@ describe('CLI run', function() {
     const url = 'chrome://version';
     const filename = path.join(process.cwd(), 'run.ts.results.json');
     const flags = getFlags(`--output=json --output-path=${filename} ${url}`);
+    flags.enableErrorReporting = false;
```

Comment:
otherwise the mock and expected errors from tests get reported to sentry, this test skips bin.ts where we do the prompting/setting to false for CI